### DISCUSSION TITLE
[ReadCSV] Add `ASSUME_NON_NULL` as an alias for `FORCE_NOT_NULL`

### DISCUSSION THREAD

**Discussion by Tishj** [duckdb/duckdb#6057] [Ideas]
I was looking at the csv reader/writer options, and had to read the description/explanation of FORCE_NOT_NULL a couple times before I knew what it did.

FORCE_NOT_NULL sounds more like a verification option to me:
- if NULL is found in the input, throw an error

But instead it's skipping NULL detection, so I feel `ASSUME_NON_NULL` fits better

**Comment by Mytherin** [comment]
I agree that the name is not extremely clear - but the reason it is called `FORCE_NOT_NULL` is because the option was copied from [Postgres](https://www.postgresql.org/docs/current/sql-copy.html).

**Reply by Tishj** [reply]
I should have guessed Postgres was to blame

**Reply by Tishj** [reply]
Then maybe just adding it as an alias instead of replacing it could be an option?

**Reply by redviking1** [reply]
Is `FORCE_NOT_NULL` valid only in the DuckDB `COPY FROM` statement?  The PostgreSQL documentation of `FORCE_NOT_NULL` for the [`COPY`](https://www.postgresql.org/docs/current/sql-copy.html) statement makes this point, but the DuckDB documentation for [`FORCE_NOT_NULL`](https://duckdb.org/docs/data/csv/overview.html#parameters) does not.

Why doesn't the DuckDB CSV reader have an option similar to `FORCE_NULL` in `COPY` for PostgreSQL?

**Reply by Mytherin** [reply]
Currently indeed `FORCE_NOT_NULL` is only accepted in the `COPY` statement. The reason for that is mostly historic - support for `FORCE_NOT_NULL` predates DuckDB's support for list types.

As to why we don't support `FORCE_NULL` - good question. We should.

**Reply by redviking1** [reply]
> Currently indeed `FORCE_NOT_NULL` is only accepted in the `COPY` statement. The reason for that is mostly historic - support for `FORCE_NOT_NULL` predates DuckDB's support for list types.

Should I create a pull request to clarify this in the DuckDB documentation of parameter `FORCE_NOT_NULL`?

> As to why we don't support `FORCE_NULL` - good question. We should.

Should I open a feature request to add parameter `FORCE_NULL`?
